src/styles/components/BlockLargeParagraph.scss

```diff
@@ -1,6 +1,14 @@
 .BlockLargeParagraph {
-  @media(max-width: 831px) {
-    margin-top: 0!important;
-    margin-bottom: 0!important;
+  @media (max-width: 831px) {
```

Comment:
We fixed basement css to not overlap the breakpoint. If you are not using `[breakpoint]-down` because of that reason.

Reply:
Oh great, yes that's why I used max-width! :)